cpp/arcticdb/version/version_core.cpp

```diff
@@ -1299,7 +1299,7 @@ VersionedItem defragment_symbol_data_impl(
             pre_defragmentation_info.append_after.value(),
             std::nullopt);
     
-    store->remove_keys(delete_keys);
+    std::ignore = store->remove_keys(delete_keys);
```

Comment:
This one is bugging me a bit. `remove_keys` returns a future, don't we want to use it/pass it to the caller.